### ISSUE TITLE
MO2 Stuck In Fullscreen

### ISSUE THREAD

**Issue by jcscrogs** [ModOrganizer2/modorganizer#531]
# The problem

Briefly describe the issue you are experiencing (or the feature you want to see added to Mod Organizer). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our discord channel

**MO2 is stuck in fullscreen for only Morrowind as I can switch to another game and it opens normally. I think the issue is with Borderless Gaming more than with MO2, as I was trying to get Morrowind borderless fullscreen, though it applied to MO2 instead, but whatever I do it doesn't release from fullscreen. This issue persists even after restarting my computer.**

# Environment

* Mod Organizer Version that exhibits the issue:

**2.1.5**

* Last Mod Organizer Verison that did not exhibit the issue (if applicable):

**Same version**

* Desktop OS/version used to run Mod Organizer:

**Windows 10, 64 bit**

# Details

If necessary, describe the problem you have been experiencing in more detail.

**NA**

# Link to Mod Organizer logs

**Attached
[logs.zip](https://github.com/ModOrganizer2/modorganizer/files/2407592/logs.zip)
**

## USVFS
Create a [GIST](https://gist.github.com) which is a paste of your USVFS logs, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

## MO Interface
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by Project579** [comment]
Why don't use the Morrowind way instead of a software?
install MGE-XE: https://www.nexusmods.com/morrowind/mods/41102/?tab=description
use this fix as well: https://community.pcgamingwiki.com/files/file/615-the-elder-scrolls-iii-morrowind-fullscreen-border-fix/

**Comment by jcscrogs** [comment]
Yes, after I did what I described above, I found the Morrowind S.T.E.P. guide and installed everything on it, which includes MGE-XE and MCP, and after messing around with MGE-XE I figured out how to do what I was originally trying to do, yet now Mod Organizer 2 is stuck in fullscreen and I don't know how to release it.

**Comment by Project579** [comment]
try closing it with task manager, the overwrite the exe with the one from the download.

**Comment by jcscrogs** [comment]
I'm not understanding, I don't know how to use task manager to do that. I now have Morrowind running perfectly, it is Mod Organizer 2 that my issue is with.

**Comment by jcscrogs** [comment]
Mod Organizer 2 is stuck in fullscreen.

**Comment by Project579** [comment]
CTRL + ALT + CANC, select Start task manager

**Comment by jcscrogs** [comment]
I know how to open Task Manager, but don't know how to use it to overwrite something. By the way, thanks for your help.

**Comment by Project579** [comment]
no use it to close MO2 process then go to the https://github.com/ModOrganizer2/modorganizer/releases and get the 7zip, use it to overwrite MO2 where the exe is: https://www.youtube.com/watch?v=CBurE8aLTxE&feature=youtu.be

**Comment by jcscrogs** [comment]
Oh, ok, I'll try this, thanks.

**Comment by jcscrogs** [comment]
I first overwrote the exe, ran it no change, then I overwrote the entire contents of the zip, ran it, and I"m still stuck in fullscreen, no change.

**Comment by Project579** [comment]
You may need to ask support on the Borderless Gaming support they probaly use their own system to make it work like that.

**Comment by jcscrogs** [comment]
Ok, the weird thing is, I can switch to a different game and it is running normally (like Fallout 4), but this is only bugged for Morrowind.

**Comment by Project579** [comment]
does your fo4 use a dedicated MO2 install with exe?

**Comment by Project579** [comment]
it might be the game that was modified, but when you ran it the program get the wrong .exe and makes MO2 fullscreen.

Also in case you are using Morrowind with MO2 I suggest you get to the Discord and get: MO2 v2.1.6 Dev Build 2 from #build .

**Comment by jcscrogs** [comment]
Yes, my fo4 uses a dedicated MO2 install.

Possibly

Ok, I'm on the Discord, but haven't gotten the Dev build.

I have to say, I love MO and MO2, I have been modding since Oblivion, and modding has come a long way since then - I find MO and MO2 to be the easiest mod manager to use and the most user friendly.

**Comment by Project579** [comment]
If you can write on Discord that is a lot faster to work with.

**Comment by jcscrogs** [comment]
Ok

**Comment by jcscrogs** [comment]
So out of curiousity, I created a new instance for Morrowind (created a new game Morrowind 2, reset the file path) and it started normally, not in fullscreen. Is there a way to import all mods just as they are in the other profile to this one?

**Comment by Project579** [comment]
Copy paste

**Comment by Al12rs** [comment]
Isn't full screen just an Ini setting? Are you sure you are editing the correct INI file? Use the Mo2 INI editor from the toolbar, the puzzle icon

**Comment by Project579** [comment]
He used a software.

**Comment by jcscrogs** [comment]
Yeah, I haven't edited the ini once.

**Comment by Al12rs** [comment]
Open the modorganizer.ini file in either appdata/local/ModOrganizer/<instance name> or in the modorganizer2 install folder. Delete all the geometry values and start Mo2 again

**Comment by Al12rs** [comment]
This isn't something we can fix on our end, so I'm closing hoping the solution above worked.

**Comment by G3Kappa** [comment]
I just wanted to point out that I had the same issue and the solution above worked. Apparently MO2 saves the window geometry when you close it, so using Borderless Gaming caused it to enter an invalid state that nonetheless got saved to disk.

If I may, I'd suggest adding a button that clears window geometry in the Workaround section of the Settings panel. It may be a rare issue, but I'm really glad that I stumbled on this thread.

**Comment by Al12rs** [comment]
Good suggestion, glad the solution worked.
Will add it to the list

**Comment by cajire** [comment]
> Good suggestion, glad the solution worked.
> Will add it to the list
Hi,
as a complete novice, I'm trying to fix this same issue but I'm a little in the dark. I'm not familiar with "Geometry lines" but I did some googling and compared what I found to my ini and don't see any. That being said, I am stuck in full screen mode :( Do you have any suggestions on this? Its been difficult since its not as cooperative when I do alt-tab to get to discord, and I cant stream if/when I need to

**Comment by Al12rs** [comment]
Go to settings -> workarounds -> reset window geometry. We added the button.

**Comment by Schaefer6** [comment]
I am trying to get borderless gaming mode to work with SSE in MO2 and I tried using the borderless gaming app (forgetting it causes this fullscreen problem with MO2) the same thing happened when I used it with Fallout 4 MO2 but with that, the solution here fixed it. **The solution above isn't fixing the issue for SSE MO2 however.** any advice? This is so frustrating and I've tried almost every solution to fix it to no avail.
